suites/fs/recovery/clusters/4-remote-clients.yaml

```diff
@@ -0,0 +1,6 @@
+roles:
```

Comment:
Also remove the 2-remote-clients.yaml file, otherwise all the tests will run twice, once with the old file and once with the new.  The teuthology test runner treats the yaml files in each dir as a list of permutations to apply to the tests.